src/main/scala/ai/privado/languageEngine/javascript/tagger/collection/CollectionTagger.scala

```diff
@@ -32,7 +33,8 @@ class CollectionTagger(cpg: Cpg, ruleCache: RuleCache) extends PrivadoParallelCp
     // TODO: Need to support more frameworks Hapijs, Koa, Loopback, Sails, Restify, Connect, AdonisJS
     val EXPRESS_CLIENT_PATTERN =
       "(?:express|fetch|@feathersjs/feathers|fastify|restify|@nestjs/cli|itty-router|koa-router|@ioc[:]Adonis|@adonisjs|@sails|sails|.*loopback).*"
-    val expressCollectionCalls = cpg.call.methodFullName(EXPRESS_CLIENT_PATTERN).l
+    val expressCollectionCalls =
+      cpg.call.methodFullName(collectionRuleInfo.patterns.headOption.getOrElse(EXPRESS_CLIENT_PATTERN)).l
```

Comment:
I would prefer the idea of not using hardCoded `Express_client_pattern`, If this pattern is not present in rules, this logic should not get executed

Reply:
Updated logic